sipssert/tasks/sipexer.py

```diff
@@ -143,9 +143,13 @@ def __init__(self, test_dir, config):
         self.logging_color = self.logging.get("color") if isinstance(self.logging, dict) else None
 
         self.target = config.get("target")
-        # self.port = config.get("port", None)
-        # if self.port:
-        #     self.port, _ = self.parse_port(self.port)
+
+        if self.template_file:
```

Comment:
This looks better, but I would check whether the path is absolute or not before forcing the mount point. At least this is what we were doing in [other tasks](https://github.com/OpenSIPS/SIPssert/blob/main/sipssert/tasks/mysql-client.py#L32)